dev-tools/mage/target/common/notice.go

```diff
@@ -133,3 +143,50 @@ func Notice() (err error) {
 
 	return nil
 }
+
+// getDependentModules returns the unique list paths of modules that the
+// github.com/elastic/elastic-agent module recursively depends on. If
+// additionalTags are specified, only files that would be compiled with
+// those build tags + "linux,darwin,windows" are examined.
+// Equivalent to running the following on the command line:
+// go list -deps -f "{{with .Module}}{{if not .Main}}{{.Path}}{{end}}{{end}}" -tags "linux,darwin,windows,{additionalTags...}"
+func getDependentModules(additionalTags ...string) ([]string, error) {
+	tags := []string{"linux", "darwin", "windows"}
+	if len(additionalTags) > 0 {
```

Comment:
check is not needed,
we could do `tags := append([]string{"linux", "darwin", "windows"}, additionalTags...)`